### PR TITLE
change year to 2018

### PR DESCRIPTION
**Pull request by pongad**

### CODE REVIEW

File: src/main/resources/com/google/api/codegen/copyright-google.txt [hunk 1]
```diff
@@ -1 +1 @@
-Copyright 2017 Google LLC
+Copyright 2018 Google LLC
```
Comment (garrettjonesgoogle) [inline src/main/resources/com/google/api/codegen/copyright-google.txt]: @lukesneeringer why did the comma disappear after the 2018 in the last update of the copyright line?

Comment (pongad) [reply src/main/resources/com/google/api/codegen/copyright-google.txt]: @garrettjonesgoogle See https://opensource.google.com/docs/releasing/preparing/, the section "Apache header".

Comment (garrettjonesgoogle) [reply src/main/resources/com/google/api/codegen/copyright-google.txt]: ugh... fine...

Comment (lukesneeringer) [reply src/main/resources/com/google/api/codegen/copyright-google.txt]: The comma should not be there. :-)

**Pull request merged**